### PR TITLE
Added Vagrantfile for isolating PostgreSQL

### PR DESCRIPTION
**Pull request by twolfson**
We have resumed working on `slack-to-trello`. In the next iteration, we are exploring allowing support for supporting multiple Slack accounts and Trello accounts. This PR sets up Vagrant since we will need some persistence for this information.

In this PR:
- Added Vagrant file
- Added a bunch of setup scripts

/cc @brettlangdon

### CODE REVIEW

**Comment by twolfson** [comment]
We are missing documentation on how to use Vagrant but that is currently due to this being in a proof of concept stage.

File: bin/bootstrap-vagrant.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  sudo apt-get install -y git
+fi
+
+# Install PostgreSQL and psycopg2 dependencies
+if ! which psql &> /dev/null; then
+  # Setup PostgreSQL 9.3 for our database
+  sudo apt-get install -y postgresql-9.3 postgresql-server-dev-9.3 python-dev
+fi
+
+# Set up PostgreSQL 9.3 configuration
```
Comment (brettlangdon) [inline bin/bootstrap-vagrant.sh:32]: Could use 9.4. Has some nice features. Better support for json, uuid (auto-gen uuids for ids, etc).

Comment (twolfson) [inline bin/bootstrap-vagrant.sh:32]: Yea but it isn't readily available on `apt` =/

**Comment by brettlangdon** [comment]
lgtm

File: bin/bootstrap-vagrant.sh (hunk 1 above)
Comment (brettlangdon) [inline bin/bootstrap-vagrant.sh:32]: Yeah, I mean, adding the official apt repo for PostgreSQL isn't too bad though.

Might mean you can even have an updated version of 9.3

But also, I don't have any issues with 9.3, just would like to get more opportunities to use it when we can.

**Comment by twolfson** [comment]
Closing this PR for now as the follow-up PR is inactive.

**Pull request closed without merging**